pkg/driver/clickhouse/clickhouse.go

```diff
@@ -119,7 +124,11 @@ func (drv *Driver) CreateDatabase() error {
 	}
 	defer dbutil.MustClose(db)
 
-	_, err = db.Exec("create database " + drv.quoteIdentifier(name))
+	if drv.onCluster() {
+		_, err = db.Exec("create database " + drv.quoteIdentifier(name) + " on cluster '{cluster}'")
```

Comment:
Forgive me for not being familiar with ClickHouse, but why is the SQL statement here `on cluster '{cluster}'`? Don't you need to specify the cluster? Where does the cluster name come from?